src/main/java/com/xceptance/neodymium/util/SelenideErrorDetailsFormatter.java

```diff
@@ -0,0 +1,31 @@
+package com.xceptance.neodymium.util;
+
+import javax.annotation.Nonnull;
+
+import com.codeborne.selenide.Driver;
+import com.codeborne.selenide.ex.SelenideErrorFormatter;
+import com.codeborne.selenide.ex.Strings;
+import com.codeborne.selenide.impl.Screenshot;
+
+public class SelenideErrorDetailsFormatter extends SelenideErrorFormatter
+{
+    @Nonnull
+    @Override
+    public String generateErrorDetails(AssertionError error, Driver driver, Screenshot screenshot, long timeoutMs)
+    {
+
+        if (Neodymium.configuration().showSelenideErrorDetails())
+        {
+            return Strings.join(super.generateErrorDetails(error, driver, screenshot, timeoutMs));
```

Comment:
return value is a string, so we probably don't need Strings.join here

Reply:
correct. removed it.